### DISCUSSION TITLE
The evaluation metrics

### DISCUSSION THREAD

**Discussion by yuhao-morning** [PaddlePaddle/PaddleSpatial#28] [BaiduKDDCup2022] [answered]
I just want to confirm that : according to the code, we need to divide the Patv predicted or truth by 1000, and then calculate the RMSE and MAE,  right ?

**Comment by havetry** [comment]
I also want to ask this question. The basic code mentions that power conversion is required, but that line of code is commented out, so whether to convert it or not?

**Comment by ramber1836** [comment, marked as answer]
Yes. 
Please refer to line 177 in metrics.py
